keystrokes/src/components/Character/CharacterDetails/CharacterDetailPage.js

```diff
@@ -127,17 +146,39 @@ function CharacterDetailPage({ userId, match }) {
 						{character.name}
 					</Grid>
 					<Grid item>
-						<Button
-							variant="contained"
-							disabled={
-								!character.specs[character.selectedSpec]
-									.configured
+						<Tooltip
+							title={
+								'Enter keybindings by clicking edit, then manual for a given layout and spec, then save it to select for gameplay!'
 							}
-							className={classes.bottomMarginNegTwo}
-							onClick={selectCharacter}
 						>
-							Select
-						</Button>
+							<span>
+								<Button
+									variant="contained"
+									disabled={
+										!(
+											allKeybindings[
+												characterKeybindings(
```

Comment:
What ever this is referencing I'd turn it into a variable.

Reply:
its a reference to state - allKeybindings are all the keybindings for a specific character. The function inside of it is a function that gets the id. Im not sure where id put the variable, since its only relevant here.

Reply:
I'd put it in your render, but that's fine just gets called twice